backend/tests/test-config.sh

```
@@ -7,8 +7,10 @@
 export TEST_API_BASE="${TEST_API_BASE:-http://localhost:7130/api}"
 
 # Admin credentials - can be overridden by environment variables
-export TEST_ADMIN_EMAIL="${TEST_ADMIN_EMAIL:-${ADMIN_EMAIL:-admin@example.com}}"
-export TEST_ADMIN_PASSWORD="${TEST_ADMIN_PASSWORD:-${ADMIN_PASSWORD:-change-this-password}}"
+
+export TEST_ADMIN_USERNAME="${TEST_ADMIN_USERNAME:-${ROOT_ADMIN_USERNAME:-admin}}"
+export TEST_ADMIN_PASSWORD="${TEST_ADMIN_PASSWORD:-${ROOT_ADMIN_PASSWORD:-password}}"
+export TEST_ADMIN_EMAIL="${TEST_ADMIN_EMAIL:-${TEST_ADMIN_USERNAME:-admin}}
```

Shellcheck (0.11.0)

[warning] Did you forget to close this double quoted string?

(SC1078)